src/Kount/Ris/Request.php

```diff
@@ -226,16 +304,13 @@ public function getResponse() {
     curl_setopt($ch, CURLOPT_VERBOSE, 0);
     curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, 2);
     curl_setopt($ch, CURLOPT_SSLVERSION, 6);
-    $this->logger->debug("Setting merchant ID custom header to RIS.");
-    curl_setopt($ch, CURLOPT_HTTPHEADER, array("X-Kount-Merc-Id: {$this->data['MERC']}"));
-
 
     // try API key authentication first, then fall back to certificates
     // which are deprecated.
     if ($this->apiKey != "") {
       $this->logger->debug("Setting API key header to RIS.");
-      curl_setopt($ch, CURLOPT_HTTPHEADER,
-        array("X-Kount-Api-Key: {$this->apiKey}"));
+      $this->logger->debug("Setting merchant ID custom header to RIS.");
+      curl_setopt($ch, CURLOPT_HTTPHEADER, array("X-Kount-Api-Key: {$this->apiKey}", "X-Kount-Merc-Id: {$this->data['MERC']}"));
```

Comment:
Adding of http headers needs to happen in one place, because the curl function CURLOPT_HTTPHEADER is called once with all the headers. A second call will overwrite the headers of the first call.